src/borg/fuse.py

```diff
@@ -260,6 +260,7 @@ def getattr(self, inode, ctx=None):
         size = 0
         dsize = 0
         if 'chunks' in item:
+            # if we would not need to compute dsize, we could get size quickly from item.size, if present.
```

Comment:
shall we just kill that? dsize reflects the space taken up in the repository and is used for st_blocks below.

we could also just say st_blocks = size / blocksize and not consider deduplication and safe the loop below for many cases.